packages/suite/src/actions/wallet/coinjoinClientActions.ts

```diff
@@ -105,7 +105,14 @@ const clientSessionPhase = (payload: CoinjoinClientEvents['session-phase']) =>
         payload,
     } as const);
 
+export const setDebugMode = (payload: CoinjoinDebugSettings) =>
```

Comment:
nit-naming: what about setDebugSettings or setCoinjoinDebugSettings instead? setDebugMode sounds quite confusing, but on the other hand it's consistent with the suiteActions.setDebugMode... so probably leave it for now and refactor later on all places

Reply:
renamed